pkg/cmd/util/helper_test.go

```diff
@@ -46,65 +39,74 @@ func (s *utilsSuite) TestProxyFields(c *check.C) {
 	// Each bit of the mask decided whether this index of `envs` would be set.
 	for mask := 0; mask <= 0b111; mask++ {
 		for _, env := range envs {
-			c.Assert(os.Unsetenv(env), check.IsNil)
+			require.Nil(t, os.Unsetenv(env))
 		}
 
 		for i := 0; i < 3; i++ {
 			if (1<<i)&mask != 0 {
-				c.Assert(os.Setenv(envs[i], envPreset[i]), check.IsNil)
+				require.Nil(t, os.Setenv(envs[i], envPreset[i]))
 			}
 		}
 
 		for _, field := range findProxyFields() {
 			idx, ok := revIndex[field.Key]
-			c.Assert(ok, check.IsTrue)
-			c.Assert((1<<idx)&mask, check.Not(check.Equals), 0)
-			c.Assert(field.String, check.Equals, envPreset[idx])
+			require.True(t, ok)
+			require.NotEqual(t, (1<<idx)&mask, 0)
+			require.Equal(t, field.String, envPreset[idx])
```

Comment:
Is it reversed here?

Reply:
the second and third params in `require.NotEqual` is reversed. Fixed.